crates/pallet-domains/src/tests.rs

```diff
@@ -186,6 +186,7 @@ parameter_types! {
     pub const DomainChainByteFee: Balance = 1;
     pub const MaxInitialDomainAccounts: u32 = 5;
     pub const MinInitialDomainAccountBalance: Balance = SSC;
+    pub const BundleLongevity: u32 = 5;
 }
```

Comment:
Why cannot we use the BlockSlotCount instead of creating a new type in config ?

Reply:
The `BlockSlotCount` needs to be greater than `BundleLongevity`, and `BlockSlotCount` is in `pallet-subspace` we need to add a config item in `pallet-domains` anyway.